### DISCUSSION TITLE
"F-16C_50 has no Pylon 10" Error

### DISCUSSION THREAD

**Discussion by GrEaSeLiTeNiN** [Khopa/dcs_liberation#1830] [Q&A] [answered]
Latest update on the DCS F-16C open beta has the HTS (Harm Targeting System) pod on pylon 5L. Using a loadout with HTS will generate the error "F-16C_50 has no Pylon 10" when pressing TAKE-OFF to generate liberation_nextturn mission file. I think "pylon 10" refers to "pylon 5L". (See screenshot). Any hotfix? Thanks
![image](https://user-images.githubusercontent.com/94265170/145354139-56a5c64c-0f70-4198-803e-b4b82afd00f0.png)

**Comment by Starfire13** [comment, marked as answer]
5.0 does not support the HTS. Remove it and the error goes away. HTS is supported in 6.0.

**Reply by GrEaSeLiTeNiN** [reply]
Yes I know, hence I was wondering if there was a hotfix before v6.0. I could of course put in the HTS at the mission editor/planner loadout.

**Reply by Starfire13** [reply]
I wouldn't say it definitely won't happen, but right now, no hotfix is planned.
